backend/src/packages/auth/auth.service.ts

```diff
@@ -18,6 +23,35 @@ class AuthService {
     return this.userService.create(userRequestDto);
   }
 
+  public async verifyLoginCredentials({
+    email,
+    password,
+  }: UserSignInRequestDto): Promise<UserSignInResponseDto | null> {
+    const user = await this.userService.findByEmail(email);
+
+    if (!user) {
+      throw new UsersError({
+        status: 404,
+        message: ExceptionMessage.USER_NOT_FOUND,
+      });
+    }
```

Comment:
this should be thrown in users service when you call findByEmail, also let's user HttpCode.NOT_FOUND instead of number

Reply:
Fixed it!